src/styles/blocks/card.scss

```diff
@@ -0,0 +1,106 @@
+.card {
+  display: flex;
+  flex-direction: column;
+
+  width: 200px;
+  height: 408px;
+  padding-top: 32px;
+
+  border: 1px solid $border-color;
+
+  &__container {
+    padding: 0 16px 16px;
+  }
+
+  &__img {
+    align-self: center;
+    width: 160px;
+    height: 134px;
+    margin-bottom: 40px;
+    img {
+      width: 100%;
+      height: 100%;
+    }
+  }
+
+  &__title {
+    font-size: 12px;
+    line-height: 1.5;
+    font-weight: 500;
+    color: $main-color;
+
+    margin-bottom: 4px;
+  }
+
+  &__code {
+    font-size: 10px;
+    line-height: 1.4;
+    font-weight: 400;
+    color: $secondary-color;
+
+    margin-bottom: 16px;
+  }
+
+  &__rating {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+
+    margin-bottom: 24px;
+  }
+
+  &__reviews {
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 1.4;
+    color: $main-color;
+  }
+
+  &__price {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+
+    margin-bottom: 16px;
+  }
+
+  &__text {
+    font-size: 12px;
+    font-weight: 400;
+    line-height: 1.5;
+    color: $secondary-color;
+  }
+
+  &__amount {
+    font-size: 16px;
+    line-height: 1.13;
+    font-weight: 700;
+    color: $main-color;
+  }
+
+  &__button {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+
+    font-size: 14px;
+    font-weight: 700;
+    color: #fff;
+    text-transform: uppercase;
+    text-decoration: none;
+
+    background-color: $button-color;
+
+    border-radius: 5px;
+    border: 1px solid $button-color;
+
+    height: 40px;
+
+    transition: all 0.2s ease-in-out;
```

Comment:
please don't use all for the transition it's bad for performance
It’s better to directly specify the property that is being animated
You can read [here ](https://www.pno.dev/articles/dont-use-the-all-keyword-in-css-transitions/)why its bad practice